formats.go

```diff
@@ -10,7 +10,7 @@ import (
 
 var (
 	labelRegex = regexp.MustCompile("^[a-z0-9][a-z0-9-_]{1,128}$")
-	nameRegex  = regexp.MustCompile("^[a-zA-Z][a-zA-Z0-9 -]{2,128}$")
+	nameRegex  = regexp.MustCompile("^[a-zA-Z0-9][a-zA-Z0-9. -]{2,128}$")
```

Comment:
you need to escape the `.` or else it will match any character, right?

Reply:
Maybe.. I tried in Regexr and when It analyzed it it treated it as the character because it was within the [ ] character list. But it would depend on the interpreter

Reply:
Ill escape it though to be sure

Reply:
also need to update the spec in this project now that its there